### DISCUSSION TITLE
debconf LXQt talk

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2278] [Media]
Just saw that: https://debconf22.debconf.org/talks/68-lxqt-team-bof/
Hopefully things are moving now (and wondering what "Bof" means...).

**Comment by edwardp3** [comment]
Is there an ETA as to when (or if?) LXQt 1.1.0 will be in Debian unstable/sid? The bug in 0.16.0 related to grouped windows is rather annoying. It had previously been reported to Debian.

Thanks.
